

export const getProfitLoss = async (req, res, next)=>{
    try {
        
    } catch (error) {
        
    }
}